File: tests/fixtures/generic/vulnerable/taint_xss.js
// @audit-fixture
// @rule: taint_xss
// @expected: detected
const express = require("express");
const app = express();
app.get("/hello", (req, res) => {
    const name = req.query.name;
    res.send("<h1>Hello " + name + "</h1>");
});

File: tests/fixtures/generic/vulnerable/taint_xss.py
# @audit-fixture
# @rule: taint_xss
# @category: security
# @expected: detected
from flask import request, Markup
@app.route("/hello")
def hello():
    name = request.args.get("name")
    return Markup(f"<h1>Hello {name}</h1>")
